src/sentry/apidocs/hooks.py

```diff
@@ -33,24 +32,32 @@ def custom_preprocessing_hook(endpoints: Any) -> Any:  # TODO: organize method,
 
 
 def custom_postprocessing_hook(result: Any, generator: Any, **kwargs: Any) -> Any:
-    for path, endpoints in result["paths"].items():
-        for method_info in endpoints.values():
-            if method_info.get("tags") is None:
-                raise UnableToProceedError(
-                    f"Please add a single tag to {path}. The list of tags is defined at OPENAPI_TAGS in src/sentry/apidocs/build.py "
+    def _check_tags() -> None:
+        if method_info.get("tags") is None:
```

Comment:
Should `method_info` be a parameter? Relying on variables in the calling scope is not as straightforward as using parameters.

Reply:
👍🏼